x-pack/plugins/actions/server/routes/update.test.ts

```diff
@@ -146,6 +149,7 @@ describe('updateActionRoute', () => {
       name: 'My name',
       config: { foo: true },
       isPreconfigured: false,
+      isDeprecated: false,
```

Comment:
Should we test at least one of the routes with a `isDeprecated: true` result?

Reply:
As we have a few of those in the end-to-end tests I didn't think it was critical to have every possible output covered in the unit test as well at route level (it *is* tested on the client itself).

Thanks Ersin!